Raven.Database/Bundles/Replication/Controllers/ReplicationController.cs

```diff
@@ -562,7 +578,14 @@ private void SaveReplicationSource(string src, string lastEtag, int batchSize, s
                                  new RavenJObject(), null);
                 });
             }
-
+            //We don't want to delay large batches of attachments otherwise we will endup with alot of data that needs to get replicated.
+            if (Request.Content.Headers.ContentLength > Database.Configuration.Replication.ReplicationPropagationDelaySizeInBytes)
+            {
```

Comment:
We are limiting batches to replication, see: https://github.com/ayende/ravendb/blob/master/Raven.Database/Bundles/Replication/Tasks/ReplicationTask.cs#L673

There is no need to handle this here.